bundles/org.openhab.binding.x/src/main/java/org/openhab/binding/x/internal/XHandler.java

```diff
@@ -82,14 +80,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     // creates list of available Actions
     @Override
     public Collection<Class<? extends ThingHandlerService>> getServices() {
-        return List.of(TwitterActions.class);
+        return Collections.singletonList(XActions.class);
```

Comment:
Why undo #15576? `List.of` is preferred because it results in less code (so you can read it faster) and it is more versatile as you can also use it to supply parameters to create immutable lists.

Reply:
Was unintentional, probably some conflict wrongly merged.